docs/contributing.md

```diff
@@ -84,3 +84,25 @@ It is often convenient to run a local version of `react-redux-firebase` within a
   **What Happened?**
 
   This error most often appears due to Webpack config. A common pattern is to provide `exclude: [/node_modules/]` setting to [babel-loader](https://github.com/babel/babel-loader), which keeps the loader from transforming code within the `node_modules` folder. Now that we have used `npm link`, your project points to your local version of `react-redux-firebase` instead of the one in your `node_modules` folder, so we have to tell the loader to also exclude transforming `react-redux-firebase`.
+
+* `Invalid hook call. Hooks can only be called inside of the body of a function component`:
+
+  **Common Solution**
```

Comment:
Nice work adding this! I have had to deal with this in the past as well. Also, I'll probably add a tidbit about how to do it if you haven't ejected from `react-scripts`

Reply:
No worries 👍